en/mavgen_c/README.md

```diff
@@ -52,6 +52,15 @@ When compiling the project, we recommend that you specify the top-level output d
 $ gcc ... -I generated/include -I generated/include/common ...
 ```
 
+In order to *send messages* you will also need to declare a variable `mavlink_system` in the **global scope**, specifying the system id and component ID of your component:
+```cpp
+mavlink_system_t mavlink_system = {
```

Comment:
Not having the '=' and adding in `-Wnarrowing` as a compiler flag can help pick bugs up before they happen.

Reply:
I don't know what you are trying to suggest here? Is this code fragment wrong? Or are you suggesting that I tell people to use `-Wnarrowing`. Sorry if this is obvious - I just copied the code in from your links.